src/test/java/eu/eutampieri/catacombs/tests/CharactersTests.java

```diff
@@ -86,4 +90,10 @@ void testBatUpdate() {
         BAT.update(10);
         // TODO implement checks
     }
+
+    @Test
+    void testBossUpdate() {
+        BOSS.update(10);
+        // TODO implement checks
+    }
```

Comment:
Not testing much, is the codecov still ok?

Reply:
cod cov passed...